### PR TITLE
feature: pump() returns last stream

### PR DESCRIPTION
**Pull request by pgte**
As discussed in #27.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ pump(source, transform, anotherTransform, dest, function(err) {
 
 If `source`, `transform`, `anotherTransform` or `dest` closes all of them will be destroyed.
 
+`pump()` returns the last stream passed in.
```
Comment (mafintosh) [inline README.md]: could we note that this is similar to how `.pipe` works?

**Comment by phated** [comment]
Please wait on this until I submit a test for this later today.

**Comment by mafintosh** [comment]
After talking to multiple developers I agree we should revert this.
Thanks for clarifying the docs @pgte - @phated you might want to include your wrapper `gulp/pump` that calls `void pump(...)`
Released in 3.0.0

**Comment by phated** [comment]
Uh, why were my test cases closed? Are you considering fixing that incorrect behavior?

**Comment by mafintosh** [comment]
I don't consider it incorrect behavior. It behaves like pipe but forwards errors to the callback. We added documentation to futher clarify this.

**Comment by phated** [comment]
Just reviewed the docs and it doesn't state that errors are swallowed if no callback is specified.

**Comment by mafintosh** [comment]
If no callback is provided they are forwarded to a noop callback and the stream pipeline it terminated. It's the users responsibility to pass one. All my modules behave this way. Fell free to PR a change that states the more clearly if you want :)

**Comment by phated** [comment]
I'd be interested to hear which developers were consulted. Everyone I've talked to about this issue don't want errors swallowed if you are using it as a replacement for pipe without that callback (as mentioned in the docs).

**Comment by mafintosh** [comment]
Added some more docs https://github.com/mafintosh/pump/commit/fee3d11cbf4c9029e1ece7a524de6ea83812b5a6. To clarify, no errors are swallowed on the returned stream. They are still emitted and forwarded to the callback if you pass one.

**Pull request closed without merging**